pandas/core/indexes/base.py

```diff
@@ -3420,7 +3439,12 @@ def _wrap_difference_result(self, other, result):
         # We will override for MultiIndex to handle empty results
         return self._wrap_setop_result(other, result)
 
-    def symmetric_difference(self, other, result_name=None, sort=None):
+    def symmetric_difference(
+        self,
+        other,
+        result_name: abc.Hashable | None = None,
```

Comment:
(None is technical not needed, it is Hashable - but it might be nice to explicitly enumerate it anyways)